src/analyzer/plugins/analyzerqueenmarybeats.cpp

```diff
@@ -9,17 +9,16 @@
 
 namespace mixxx {
 namespace {
-// These are the preferred window/step sizes from the BeatTrack VAMP plugin.
-constexpr size_t kWindowSize = 1024;
-constexpr size_t kStepSize = 512;
 
-DFConfig makeDetectionFunctionConfig() {
+constexpr float kStepSecs = 0.01161; // 512 samples at 44100
```

Comment:
Please add comments why we use this rather arbitrary number of seconds (that nicely maps to 512 sample at 44.1 kHz only) instead of the fixed step and window sizes independent of the sample rate.

Reply:
done!